app/cdap/components/PipelineList/DraftPipelineView/DraftActions/index.tsx

```diff
@@ -46,25 +48,39 @@ class DraftActions extends React.PureComponent<IProps, IState> {
 
   private handlePipelineExport = () => {
     const draft = this.props.draft;
-
     this.pipelineConfig = {
       name: draft.name,
       description: draft.description,
       artifact: draft.artifact,
       config: draft.config,
     };
 
-    if (window.Cypress) {
-      this.openExportModal();
-      return;
-    }
-    const postExportCb = () => {
-      this.pipelineConfig = {};
-      this.setState({ showPopover: false });
+    const params = {
+      context: getCurrentNamespace(),
+      draftId: draft.id,
     };
-
-    // Unless we are running an e2e test, just export the pipeline JSON
-    downloadFile(this.pipelineConfig, postExportCb);
+    MyPipelineApi.getDraft(params).subscribe({
+      next: (res: any) => {
+        this.pipelineConfig = { ...this.pipelineConfig, config: res.config };
+      },
+      error: (err) => {
+        // tslint:disable-next-line:no-console
+        console.log('Unable to fetch draft details', err);
```

Comment:
Shouldn't we find a better way to display the error to the user (banner or alert on the UI)? This is like silently failing.

Reply:
made some changes